### PR TITLE
Updated: fixed sb runtime callout as per feedback.

### PR DESCRIPTION
**Pull request by rhoads-zach**
@inoxx03 please take a look and merge this when you have a chance.

### CODE REVIEW

File: docs/spring-boot-runtime/master.adoc [hunk 1]
```diff
@@ -10,8 +10,8 @@ This guide covers concepts as well as practical details needed by developers to
 
 include::topics/sb-tomcat-runtime-overview.adoc[leveloffset=+1]
 
+{SpringBoot} has a version of its runtime that is tested and verified to run on OpenShift. Some components, like the Embedded Tomcat Container and Java container image, when used with {SpringBoot} are provided as part of a Red Hat subscription.
```
Comment (tradej) [inline docs/spring-boot-runtime/master.adoc]: The word order makes the second sentence difficult to read, IMO. I would rephrase as:

Some components—for example the Embedded Tomcat Container and the Java container image—are provided as part of a Red Hat subscription when used with {SpringBoot}.

**Comment by inoxx03** [comment]
@rhoads-zach I see that @tradej has left you some comments. I propose merging this now and I'll file an issue for the changes that we can do in a separate PR. WDYT?

**Comment by tradej** [comment]
@inoxx03 Since Zach's out, let's do that.

**Comment by inoxx03** [comment]
agreed:) I'll merge this shortly.

**Comment by inoxx03** [comment]
merged: 3725bc8ecad734689a379d18349d5530f34ce403

**Comment by inoxx03** [comment]
follow-up issue filed: #463

**Comment by lavernw1** [comment]
Yes I like @tradej 's update too.
